graphql/schema.graphql

```diff
@@ -146,7 +146,39 @@ input CoursePreferenceInput {
 input GenerateScheduleInput {
   year: Int!
   term: Term!
-  courses: CourseID!
+  courses: [CourseInput!]
+  algorithm1: Company!
+  algorithm2: Company!
+}
+
+input EditCourseInput {
```

Comment:
Shouldn't there be some attribute to identify a course? Looks like just `sectionNumber` which I assume is A01, etc, but how do we know what course to edit in the mutation?

Reply:
Could add the `CourseInput` type perhaps and then remove `sectioNumber` since it's a part of course input

Reply:
I dont think we should remove sectionNumber. As sectionNumber is not a part of CourseInput but section is.

Reply:
Gotcha